build.gradle.kts

```diff
@@ -20,6 +20,7 @@ recipeDependencies {
     parserClasspath("org.mockito:mockito-all:1.10.19")
     parserClasspath("org.mockito:mockito-core:3.+")
     parserClasspath("org.jmockit:jmockit:1.49")
+    parserClasspath("org.jmockit:jmockit:1.22")
```

Comment:
You can't add another version of the same library on the classpath. JMockit 1.49 is the last published version from 2019 so it should be the most appropriate choice.

Reply:
@timtebeek do you know how this version will get picked in the `ChangeType` recipe added in jmockit.yml? I can easily see how using `classpathFromResources` allows us to explicitly choose the classpath dependencies, but I'm unclear on this case.

Reply:
We do a prefix match, that you can extend to include the version number like `jmockit-1.22` if you want to be specific. Rewrite-spring has the most variation in acceptable patterns if you want to [see some examples](https://github.com/search?q=repo%3Aopenrewrite%2Frewrite-spring%20classpathFromResources&type=code).